sway/desktop/output.c

```diff
@@ -513,7 +513,10 @@ int output_repaint_timer_handler(void *data) {
 		return 0;
 	}
 
-	output->wlr_output->block_idle_frame = false;
+	output->wlr_output->frame_pending = false;
+
+	bool surface_needs_frame = output->surface_needs_frame;
+	surface_needs_frame = false;
```

Comment:
Seems like a left-over debug line which actually disables the behavior?

Reply:
Ah, my latest push had not gone through. Weird.

The correct form is `output->surface_needs_frame = false`.